python/grass/temporal/aggregation.py

```diff
@@ -316,6 +317,47 @@ def aggregate_by_topology(
         if "overlapped" in topo_list and granule.overlapped:
             for map_layer in granule.overlapped:
                 aggregation_list.append(map_layer.get_name())
+        if "related" in topo_list:
+            aggregation_weights = []
+            set_list = set()
+            try:
+                for map_layer in granule.overlaps:
+                    set_list.add(map_layer)
+            except:
+                pass
```

Comment:
This seems to be similar to the code above, but it is written in a completely different style. Any reason not to follow the above style? Additionally, the try-except is unclear as it is written now since it is a bare except. See Flake8: `./temporal/aggregation.py:326:13: E722 do not use bare 'except'` (also in CI).